Formula/amr-client.rb

```diff
@@ -0,0 +1,20 @@
+
+class AmrClient < Formula
+  desc "Ask Mr. Robot Simulator Client"
+  homepage "http://www.askmrrobot.com/"
+  url "https://static2.askmrrobot.com/amrbeta/client/AskMrRobotClient-159-osx.10.11-x64.zip"
```

Comment:
We don't accept closed-source software; you may want to consider hosting this in your own [tap](https://github.com/Homebrew/brew/blob/master/docs/How-to-Create-and-Maintain-a-Tap.md). Thanks!